packages/components-js/projects/partials/scripts/generateInitialStylesPartial.ts

```diff
@@ -75,6 +75,19 @@ type GetInitialStylesOptionsWithoutTags = Omit<GetInitialStylesOptions, 'format'
           outlineColor: 'transparent',
         },
       },
+
+      // the following selectors don't work within ::slotted() pseudo selector, therefore we have to apply them via light DOM
+      'input::-webkit-outer-spin-button, input::-webkit-inner-spin-button, input[type="search"]::-webkit-search-decoration, input[type="search"]::-webkit-search-cancel-button':
```

Comment:
This is different from how the styles previously were in the component? Is that on purpose?